gym/core.py

```diff
@@ -341,5 +354,21 @@ def _configure(self, *args, **kwargs):
     def _seed(self, seed=None):
         return self.env.seed(seed)
 
+    @property
+    def wrappers(self):
```

Comment:
Let's remove `wrappers` and `name` (user can build that list themselves, and the names on their own aren't necessarily meaningful since people can instantiate with various arguments).